packages/wallet/src/redux/protocols/indirect-defunding/reducer.ts

```diff
@@ -44,14 +46,22 @@ export const initialize = (
       };
     }
     newSharedData = signResult.store;
+    // send a request for opponent to start new defunding process first, because they may not yet have done so
+
+    const actionToRelay = defundRequested({
+      processId: '0XTODO', // wish to terminate existing process in opponent's wallet (could use currentProcessId but this is not on SharedData)
+      channelId,
+    });
+
+    const messageRelay0 = messageRelayRequested(theirAddress(ledgerChannel), actionToRelay);
```

Comment:
could we rename `messageRelay0` to something more meaningful like `defundMessageRelay`?

Reply:
:+1: addressed in https://github.com/magmo/apps/pull/530/commits/4e1fc8a2317d5098fed7a54cc0efc5b71455fb6a